src/components/Layout/Footer.jsx

```diff
@@ -22,6 +22,9 @@ function Footer() {
           backgroundColor: 'primary.main',
```

Comment:
at some point the screen is not responsive . 
![image](https://user-images.githubusercontent.com/29543196/210779464-86e93499-5967-4090-a3d1-1345e0531358.png)
can we fix that ?